macOS/DuckDuckGo/NavigationBar/View/AddressBarViewController.swift

```diff
@@ -488,12 +491,33 @@ final class AddressBarViewController: NSViewController {
 
     // MARK: - Layout
 
+    /// Workaround for macOS 26.0 NSTextFieldSimpleLabel rendering bug
+    /// Aggressively hides internal label views that incorrectly remain visible
+    @available(macOS 26.0, *)
+    private func forceHideInternalTextFieldLabels(in textField: NSTextField) {
```

Comment:
since the AddressBarTextField is our custom class, maybe we could just override its `addSubview:` and filter out the `NSTextFieldSimpleLabel`-s so it doesn't get added?